### PR TITLE
Comment types

### PR DESCRIPTION
**Pull request by kevinpapst**
Changes proposed in this pull request:
- removing duplicated code
- new view helper to render the comment type select-box
- remove $commentTypes from global namespace

Reasons for this pull request:
- code cleanup

### CODE REVIEW

File: templates/helpers/CommentTypeSelect.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Kimai; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * A select box to choose the comment type.
+ *
+ * @author Kevin Papst
```
Comment (simonschaufi) [inline templates/helpers/CommentTypeSelect.php:23]: i would remove this

File: templates/helpers/CommentTypeSelect.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ */
+class Zend_View_Helper_CommentTypeSelect extends Zend_View_Helper_FormSelect
+{
+    /**
+     * @param null $value
```
Comment (simonschaufi) [inline templates/helpers/CommentTypeSelect.php]: type is string or int, not null

File: templates/helpers/CommentTypeSelect.php (hunk 1 above)
Comment (kevinpapst) [inline templates/helpers/CommentTypeSelect.php:23]: we have to discuss this. its all over the code and i don't mind this for new files (just find it a bit overwhelming for each method, like in mysql class).
I am going to start a section in the documentation about coding standards ...

Comment (simonschaufi) [inline templates/helpers/CommentTypeSelect.php:23]: what do you want to achieve with this? git can show me the author of every line. we don't need that extra information.

Comment (kevinpapst) [inline templates/helpers/CommentTypeSelect.php:23]: credits to whom they belong. besides that: why not?

Comment (simonschaufi) [inline templates/helpers/CommentTypeSelect.php:23]: we are a team, not single devs. this is **our** code ;)

**Pull request merged**